scripts/init_mongo.js

```diff
@@ -3,19 +3,4 @@ const MongoClient = require('mongodb').MongoClient;
 MongoClient.connect('mongodb://localhost', (err,client) => {
     const db = client.db('WhiteboardDB');
     db.collection('whiteboards').remove({}); //Clear the database to ensure we have a fresh start
-    db.collection('whiteboards').insert(
-        {
-            _id: "1",
-            URI: "12345"
```

Comment:
Good call on getting rid of the mock data.

Reply:
Agreed.